uint8 dxlID
float64 setData
---
bool complete
